style/gallery.less

```diff
@@ -177,7 +177,7 @@
 					height: 44px;
 					width: 44px;
 					display: flex;
-					background-image: inline-svg( '../images/attribution-icon-info.svg' );
+					background-image: url( https://wikipedia.org/static/images/wikipedia-preview/attribution-icon-info.svg );
```

Comment:
The canonical domain name is `www.wikipedia.org`, avoid redirects or staleness.

Reply:
Thanks, will update it